vpc/tool/assign3/assign_network.go

```
@@ -75,6 +75,7 @@ func Assign(ctx context.Context, instanceIdentityProvider identity.InstanceIdent
 			err = errors.Wrap(writeError, err.Error())
 			return err
 		}
+		return err
```

whoa whoops? I guess we were swallowing this error before?